pkg/tags/tags_test.go

```diff
@@ -39,7 +39,7 @@ func TestParseTags(t *testing.T) {
 			2,
 		},
 		{
-			" store forever =false&factory=true",
+			"_store_forever_=false&factory=true",
```

Comment:
What is this change about?

Reply:
The test was failing, so I noticed that fixed it.